apis/infix-en_GB/atrium-api-infix-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/pathAssertions.kt

```diff
@@ -295,3 +297,52 @@ val <T : Path> Expect<T>.extension: Expect<String>
  */
 infix fun <T : Path> Expect<T>.extension(assertionCreator: Expect<String>.() -> Unit): Expect<T> =
     ExpectImpl.path.extension(this).addToInitial(assertionCreator)
+
+/**
+ * Expects that the subject of the assertion (a [Path]) has the same textual content
+ * as [targetPath].
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.13.0
+ */
+infix fun <T : Path> Expect<T>.hasSameTextualContentAsDefaultArgs(
```

Comment:
Here you can use the following
```suggestion
infix fun <T : Path> Expect<T>.hasSameTextualContentAs(
```
I meant `hasSameTextualContentAsDefaultArgs` only for the spec

Reply:
Please rename it to `hasSameTextualContentAs` again and workaround the type inference problems in the Spec.
For api-fluent, use an indirection as you already did (i.e. a function in the companion) since 
`hasSameTextualContentAs` in api-fluent expects 4 arguments instead of 1. 
However, in api-infix you have an overload which expects only 1 argument so you can use fun1. You need to help Kotlin as the current type inference is not good enough. I made a suggestion in the spec